documentation/docs/09-seo.md

```diff
@@ -0,0 +1,72 @@
+---
+title: SEO
+---
+
+The most important aspect of SEO is to create high-quality content that is widely linked to from around the web. However, there are a few technical considerations for building sites that rank well.
+
+### Out of the box
+
+#### SSR
+
+While search engines have got better in recent years at indexing content that was rendered with client-side JavaScript, server-side rendered content is indexed more frequently and reliably. SvelteKit employs SSR by default, and while you can disable it in [`handle`](/docs/hooks#handle), you should leave it on unless you have a good reason not to.
+
+> SvelteKit's rendering is highly configurable and you can implement [dynamic rendering](https://developers.google.com/search/docs/advanced/javascript/dynamic-rendering) if necessary. It's not generally recommended, since SSR has other benefits beyond SEO.
+
+#### Performance
+
+Search engines take site performance into account as a ranking factor. [Web Vitals](https://web.dev/vitals/) is a term coined by Google to provide performance metrics regarding a site's load speed. [Google takes the Core Web Vitals into account](https://developers.google.com/search/blog/2020/11/timing-for-page-experience) in deciding which sites to show at the top of the rankings.
+
+SvelteKit is optimized to give great scores out-of-the-box. Check out [how this site performs on Google PageSpeed Insights](https://pagespeed.web.dev/report?url=https%3A%2F%2Fkit.svelte.dev%2F). We invite you to compare against sites built with other frameworks. If scores on your site are not high, we recommend trying to minimize the size of included dependencies and to use the developer tools provided by your browser to diagnose.
+
+#### Normalized URLs
+
+SvelteKit redirects URLs with trailing slashes to versions without trailing slash or vice versa depending on your [configuration](configuration#trailingslash).
+
+### SEO setup
+
+#### `meta` tags
+
+It is generally recommended to add `meta` HTML tags to your page. These can be created just as any other Svelte element. The Svelte Society website has an [example](https://github.com/svelte-society/sveltesociety.dev/blob/staging/src/lib/components/Seo.svelte). You may also find the [`stuff`](loading#input-stuff) feature as helpful for constructing `meta` tags in some cases where you want to pass content from a page to a layout.
+
+#### Structured data
+
+There is not too much special you need to do in SvelteKit to provide structured data, so we recommend that you reference [Google's documentation regarding structured data](https://developers.google.com/search/docs/advanced/structured-data/intro-structured-data). If you're using `svelte-preprocess`, please note that [you'll currently need to use the `preserve` option](https://github.com/sveltejs/svelte-preprocess/issues/305):
+
+```
```

Comment:
don't think i can add a commit suggestion with this going haywire, but the block should have a `js`